dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -0,0 +1,1551 @@
+      {
+       "value": "assigning_work",
+       "text": "Assigning new work"
+      },
+      {
+       "value": "materials_for_indpendent_work",
+       "text": "Finding materials that students can work on independently"
+      },
+      {
+       "value": "login_info",
+       "text": "Getting students their login information"
+      },
+      {
+       "value": "materials_for_offline",
+       "text": "Finding materials to work on that don't require internet and device access"
+      },
+      {
+       "value": "communicating",
+       "text": "Communicating with my students while they are at home"
+      },
+      {
+       "value": "instruction_new_concepts",
+       "text": "Providing instruction on new concepts to my students"
+      },
+      {
+       "value": "student_collaboration",
+       "text": "Finding ways for students to collaborate"
+      },
+      {
+       "value": "lms_integration",
+       "text": "Lack of integration with my school’s LMS"
+      }
+     ]
+    },
+    {
+     "type": "comment",
+     "name": "one_challenge_FR_remote",
+     "title": "For one of the items you identified as a huge challenge, please describe any ideas you have for how code.org might address it.",
+     "cols": 40,
+     "rows": 6
+    },
+    {
+     "type": "matrix",
+     "name": "cdo_strategies_remote",
+     "title": "When teaching {course} remotely how often did YOU explicitly do the following?",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Never"
+      },
+      {
+       "value": "2",
+       "text": "Rarely<br>(a few times a year)"
+      },
+      {
+       "value": "3",
+       "text": "Sometimes<br>(once or twice a month)"
+      },
+      {
+       "value": "4",
+       "text": "Often<br>(once or twice a week)"
+      },
+      {
+       "value": "5",
+       "text": "Almost every lesson"
+      }
+     ],
+     "rows": [
+      {
+       "value": "pair_programming",
+       "text": "Ask students to write code with another student?"
+      },
+      {
+       "value": "peer_feedback",
+       "text": "Ask a student to give feedback on a classmate’s work?"
+      },
+      {
+       "value": "think_pair_share",
+       "text": "Ask a student to share their ideas with classmates in small group or large group discussion?"
+      },
+      {
+       "value": "normalize_failure",
+       "text": "Remind students that failure is a normal and natural part of the problem solving process."
+      },
+      {
+       "value": "journaling",
+       "text": "Gave students the opportunity to write their thoughts down in a journal or on paper."
+      }
+     ]
+    },
+    {
+     "type": "matrix",
+     "name": "facilitation_student_interest_remote",
+     "title": "When teaching {course} remotely how often did YOU explicitly do the following?",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Never"
+      },
+      {
+       "value": "2",
+       "text": "Rarely<br>(a few times a year)"
```

Comment:
<br> won't render as a line break (I don't think there's a way to put line breaks here)